Exam/CT/src/exercice3/model/Board.java

```diff
@@ -45,6 +45,48 @@ public Board(int nbRows, int nbColumns) {
 		}
 		return c;
 	}
+	
+	public int computeNeighboringBomb(int row, int col) {
```

Comment:
Glad to see you tried to take this one on but... you could have simply re-use `getNeighboringCells` directly.